sources/packages/web/src/services/ConfirmationOfEnrollmentService.ts

```diff
@@ -4,7 +4,7 @@ import {
   ApplicationDetailsForCOEAPIOutDTO,
   COEDeniedReasonAPIOutDTO,
   COESummaryAPIOutDTO,
-  ConfirmationOfEnrollmentAPIInDTO,
+  ConfirmEnrollmentOptions,
```

Comment:
why `APIInDTO,` is removed?

Reply:
It is confusing having this is the DTO context when it is not a DTO. For the API I recommended moving it to a `models` file and here we can move also, IMO.